api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -533,9 +533,9 @@ public DbUser syncComplianceTrainingStatusV2(DbUser dbUser, Agent agent)
           (badgeName) -> {
             Optional<BadgeDetailsV2> badge =
                 Optional.ofNullable(userBadgesByName.get(badgeName))
-                    .filter(BadgeDetailsV2::getValid);
+                    .filter(BadgeDetailsV2::isValid);
 
-            if (badge.isEmpty()) {
+            if (!badge.isPresent()) {
```

Comment:
can revert - this conflict is due to my recent PR

Reply:
Thank you! I was wondering why I changed it since I like the "original" better.